tests/entities/test_listeners.py

```diff
@@ -66,6 +67,23 @@ def test_fi_history_listener(self, mocker: MockerFixture):
         self.fi_history.insert.assert_called_once()
         self.mapping_history.insert.assert_called_once()
 
+    def test_fi_history_listener_no_types(self, mocker: MockerFixture):
+        inspect_mock = mocker.patch("regtech_user_fi_management.entities.listeners.inspect")
+        attr_mock1: AttributeState = Mock(AttributeState)
+        attr_mock1.key = "name"
+        attr_mock2: AttributeState = Mock(AttributeState)
+        attr_mock2.key = "event_time"
+        state_mock: InstanceState = Mock(InstanceState)
+        state_mock.attrs = [attr_mock1, attr_mock2]
+        inspect_mock.return_value = state_mock
+        fi_listener = _setup_fi_history(self.fi_history, self.mapping_history)
+        no_types = deepcopy(self.target)
+        delattr(no_types, "sbl_institution_types")
```

Comment:
Is everything identical to the above test with the exception of these two lines? It _seems_ like it, but my eyeball 👁️  `diff` is only so good.

Reply:
yeap, only removed the types from the dao

Reply:
...and is deleting the `sbl_institutions_types` attribute from the object really simulating what will happen in the case? Is it not enough to just make it `None` or an empty list?

Reply:
yeap, empty list works, actually that probably simulates the actual flow better, I'll update that; setting None makes it incompatible type.